eclass-writing/text.xml

```diff
@@ -327,6 +350,53 @@ variables are as follows:
     this tag. Must appear as the first tag in the comment block.
   </ti>
 </tr>
+<tr>
+  <ti>
+    <c>@PRE_INHERIT</c>
+  </ti>
+  <ti>
+    YES
+  </ti>
+  <ti>
+    <d/>
+  </ti>
+  <ti>
+    This tag describes whether the variable must be defined before
+    the eclass is inherited. This is important if any e.g. dependencies
+    are modified in global scope at the point of sourcing.
+  </ti>
+</tr>
+<tr>
+  <ti>
+    <c>@USER_VARIABLE</c>
+  </ti>
+  <ti>
+    YES
+  </ti>
+  <ti>
+    <d/>
+  </ti>
+  <ti>
+    This tag describes whether the variable is unsuitable for use in ebuilds,
+    i.e. if it is solely for user consumption via e.g. make.conf or a similar
+    mechanism.
+  </ti>
+</tr>
+<tr>
+  <ti>
+    <c>@OUTPUT_VARIABLE</c>
+  </ti>
+  <ti>
+    YES
+  </ti>
+  <ti>
+    <d/>
+  </ti>
+  <ti>
+    This tag indicates the variable's value (which will be set by the eclass)
```

Comment:
I think adding "that" makes the sentence less ambiguous:
> This tag indicates _that_ the variable's value (which will be set by the eclass) should be read within an ebuild.

When I first read the existing sentence, I thought that it meant "This tag indicates the variable's value".

Reply:
Agreed, thank you!